modules/administration_manual/pages/configuration/files/encryption/master-key-encryption.adoc

```diff
@@ -0,0 +1,127 @@
+[[master-key-based-encryption]]
+= Master Key Based Encryption
+Matthew Setter <matthew@matthewsetter.com>
+:experimental:
+:keywords: master key based encryption, encryption
+:description: This guide will show you how to work with Master Key encryption in ownCloud. 
+
+With Master Key based encryption, there is only one key (or key pair) and all files are encrypted using that key pair. 
+This is highly recommended for new instances to avoid restrictions in functionality of user key encryption.
+
+== Enabling Master Key Based Encryption
+
+To enable Master Key based encryption requires two steps:
+
+. xref:enable-the-encryption-app[Enable the encryption app]
+. xref:enable-and-configure-master-key-based-encryption[Enable and configure User-Key based Encryption]
+
+[TIP] 
+====
+We strongly encourage you to put your server in single user mode before setting up encryption.
+
+To do so, run the following command:
+
+....
+sudo -u www-data php occ maintenance:singleuser --on
+....
+====
+
+[[enable-the-encryption-app]]
+=== Enable the Encryption App
+
+include::enable-encryption.adoc[leveloffset=+1,tag=enable-encryption-app-via-command-line]
+
+[[enable-and-configure-master-key-based-encryption]]
+=== Enable and Configure Master Key Based Encryption
+
+To enable and configure Master Key based encryption via the command-line, involves several commands.
+
+. Enable encryption.
+. Enable the master key.
++
+NOTE: The master key mode has to be setup in a newly created instance.
```

Comment:
Question, what does that mean?
Can you enable it on an instance that runs already fo a while and if yes what is the impact?

Reply:
I've incorporated your text below, but am leaving this, to allow for multiple installation configurations.